src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -287,6 +296,49 @@ export class NetCoreTaskHelper implements TaskHelper {
 
         return volumes;
     }
+
+    private async buildWithDotnetSdk(context: DockerBuildTaskContext, buildDefinition: DockerBuildTaskDefinition): Promise<void> {
+        //reference: https://learn.microsoft.com/en-us/dotnet/core/docker/publish-as-container
+        const publishFlag = NetCoreTaskHelper.isWebApp ? '-p:PublishProfile=DefaultContainer' : '/t:PublishContainer';
+        const [runtimeOs, runtimeArch] = this.getOsAndArch(buildDefinition);
+        const [imageName, imageTag] = this.getImageNameAndTag(buildDefinition);
+
+        const sdkBuildCommand = `dotnet publish --os ${runtimeOs} --arch ${runtimeArch} ${publishFlag} -c Release -p:ContainerImageName=${imageName} -p:ContainerImageTag=${imageTag}`;
+        await context.terminal.execAsyncInTerminal(
+            sdkBuildCommand,
+            {
+                folder: context.folder,
+                token: context.cancellationToken,
+            }
+        );
+    }
+
+    private getImageNameAndTag(buildDefinition: DockerBuildTaskDefinition): [string, string] {

```

Comment:
Consider using the existing parseDockerLikeImageName method as it handles a number of image name/tag format edge cases.

Reply:
Will do